pkg/agent/agent.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	envoyMetrics "github.com/datawire/ambassador/v2/pkg/api/envoy/service/metrics/v3"
+	envoyMetrics "github.com/datawire/ambassador/v2/pkg/api/envoy/service/metrics/v2"
```

Comment:
Hmmm, I haven't looked at this part of Envoy in a bit – the final `v2` in the package here is not the Envoy API version, IIRC, but the version number for sending the metrics to the metrics service, right?

Reply:
Exactly, @kflynn.

Reply:
I'm timeboxing a few minutes to try to upgrade this to v3, it'll be better to start at a more recent version.

Reply:
Upgraded to the transport api v3 for metrics in [0a1fccb](https://github.com/emissary-ingress/emissary/pull/4053/commits/0a1fccb65dd241913bfb5252236c07e561988082). We should be more future-proof now. 👍